### ISSUE TITLE
CSS: Basic-properties

### ISSUE THREAD

**Issue by pixeline** [becodeorg/free2code#9]
This zone feels not ready to me and I am not sure what are the learning objectives here.
@NickBanken  Can you list them here in a comment ? I can then create the content according to the LO

**Comment by pixeline** [comment]
oh i got it! the exercises are in the html :-)

**Comment by NickBanken** [comment]
@pixeline Yes, maybe I should of written some clear instructions in markdown...
My bad! :)

**Comment by pixeline** [comment]
nah it's cool like this !
